apps/guardian-ui/src/components/SetConfiguration.tsx

```diff
@@ -75,6 +72,9 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
   });
   const [mintAmounts, setMintAmounts] = useState<number[]>([]);
   const [error, setError] = useState<string>();
+  const [numPeers, setNumPeers] = useState(
+    stateNumPeers ? stateNumPeers.toString() : isSolo ? '1' : '4'
```

Comment:
Why are we hard coding four? Can we define a const or function for this instead?

Reply:
the hardcode is what it starts with for if they're a host, so in their meta configuration it starts at 4